sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/CosmosEncryptionAsyncContainer.java

```diff
@@ -99,6 +99,38 @@ EncryptionProcessor getEncryptionProcessor() {
         return this.encryptionProcessor;
     }
 
+    /**
+     * create item and encrypts the requested fields
```

Comment:
Lets change the doc of create Item apis inline with cosmos core sdk . In new apis and existing one as well
Link below 
   ```
 /**
     * Creates an item and encrypts the requested fields.
     * <p>
     * After subscription the operation will be performed. The {@link Mono} upon
     * successful completion will contain a single resource response with the
     * created Cosmos item. In case of failure the {@link Mono} will error.
     *
     * @param <T> the type parameter.
     * @param item the Cosmos item represented as a POJO or Cosmos item object.
     * @return an {@link Mono} containing the single resource response with the
     * created Cosmos item or an error.
     */
```

Reply:
Fixed